### PR TITLE
chapter 9 + images

### PR DESCRIPTION
**Pull request by defuneste**
For chapter 9! I corrected few spelling but this is a huge chapter

### CODE REVIEW

File: 09.Rmd [hunk 1]
```diff
@@ -2,12 +2,500 @@
...
+      - broom.mixed::tidy()
+      - tidybayes::add_fitted_draws()
+      - tidybayes::add_predicted_draws()
+
+5. Posterior predictions
+
+    a. By hand (dplyr mutate and summarize)
+    b. `rstanarm::posterio_predict()` + `rstantools::posterior_interval()`  
+
+## Ressources:
```
Comment (jonthegeek) [inline 09.Rmd]:
```suggestion
## Resources: 
```

**Review by jonthegeek — CHANGES_REQUESTED** [review]
I found one small typo. There might be others but I see this one!

We'll also watch to see if the build works. If it doesn't, we'll have to make more changes.

**Review by jonthegeek — APPROVED** [review]
If the build works this looks good!

**Comment by defuneste** [comment]
thx Jon!

**Pull request merged**